tools/dockerfile/interoptest/grpc_interop_java/Dockerfile_oracle8

```diff
@@ -0,0 +1,73 @@
+# Copyright 2015, Google Inc.
```

Comment:
For all dockerfiles:

We actually use templates to generate our docker files to make sure slightly different versions of our dockerfiles stay in sync (e.g.   openjdk8 and oracle8 should be almost identical, with just the installed java package being different - and a template should be used to make sure that happens).

See templates here:
https://github.com/grpc/grpc/tree/master/templates/tools/dockerfile/interoptest

tools/buildgen/generate_projects.sh is used to regenerate files from templates.